components/raftstore-v2/src/operation/command/mod.rs

```diff
@@ -425,6 +441,25 @@ impl<EK: KvEngine, R: ApplyResReporter> Apply<EK, R> {
                     return Err(box_err!("invalid admin command type"));
                 }
             };
+            // Check if the region epoch changes as expected
```

Comment:
This should be a unit test instead of code for production.

Reply:
Removed